### ISSUE TITLE
Workspaces - Keep services active

### ISSUE THREAD

**Issue by danpaulson** [meetfranz/franz#1414]
**Is your feature request related to a problem? Please describe.**
When I swap Workspaces, all of the services have to reconnect. This mostly defeats the point of using workspaces.

**Describe the solution you'd like**
Workspace services are kept active even if the workspace isn't active, or an option to enable this.

**Describe alternatives you've considered**
The alternative is to simply not use workspaces (and thus not use premium!)

**Comment by DominikGuzei** [comment]
Thanks for the feedback @danpaulson, we will discuss this as an optional!

**Comment by derekmhewitt** [comment]
I completely agree, I recently became a supporter mostly for Workspaces (and to support the project) and the amount of transition waiting I have to do between changing Workspaces defeats the purpose of the feature.

**Comment by Wouter0100** [comment]
Would also love to see this. I became a supporter because I expected that it was being used like this, and that notifications still come through.

**Comment by Wouter0100** [comment]
I created the feature myself and offered it to Franz in PR #1498. Hopefully it gets accepted :).

**Comment by timvahlbrock** [comment]
I'm happy to see, that this feature might be available in future versions of franz, but I may have another suggestion: 
What about adding another option in the service or workspace settings to keep only that specific service/worspace loaded?
I don't know how resource intensive keeping all workspaces loaded is, so maybe it would be useful for setups that contain a large number of services to keep only important services/workspaces loaded.
